mitxgraders/baseclasses.py

```diff
@@ -205,6 +207,60 @@ def log_output(self):
         content = "\n".join(self.debuglog)
         return "<pre>{content}</pre>".format(content=content)
 
+    @classmethod
+    def ensure_text_inputs(cls, student_input, allow_lists=True, allow_single=True):
+        """
+        Ensures that student_input is a list of text strings.
+
+        NOTE: Overriden by ItemGrader to ensure input is a single text string.
```

Comment:
Add a note for `ListGrader` too?

Reply:
👍 
I'm also changing staticmethod to classmethod since `cls` is never used.

(I had erroneously thought staticmethods couldn't be called by super.)